README.adoc

```diff
@@ -189,4 +189,4 @@ added after the original pull request but before a merge.
   other target branch in the main project).
 * When writing a commit message please follow http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html[these conventions],
   if you are fixing an existing issue please add `Fixes gh-XXXX` at the end of the commit
-  message (where XXXX is the issue number).
\ No newline at end of file
+  message (where XXXX is the issue number).
```

Comment:
Actually this change should be done in the README inside docs module - this README is generated from that code